reference/types/floating_point_number.md

```diff
@@ -10,15 +10,42 @@ Programming languages may name their floating point type to the amount of _preci
 - Quadruple (occupying 128 [bits][type-bit])
 - Octuple (occupying 256 [bits][type-bit])
 
-# Implementations
+## What to cover
 
-- [C# (basic)][implementation-csharp-basic]
-- [C# (advanced)][implementation-csharp-advanced]
+A student may have no understanding of floating point numbers as more than numbers with decimal points.
+
+- Explain that a floating point number is not *just* a number with a decimal place.
+- Explain what floating point types are used in your langauge, and when to use them.
+- Ensure that type conversations are understood if appropriate (e.g. rounding, precision-changing)
```

Comment:
"type conversations". Is that the phrase to use? I'm not versed in this area. Are rounding and precision-changing considered type conversations? I though they were just changing the value.

Reply:
And is that meant to be "type conversations" or "type conversions"?

Reply:
```suggestion
- Ensure that type conversions are understood if appropriate (e.g. rounding, precision-changing)
```